sdk/python/jobs/automl-standalone-jobs/automl-forecasting-github-dau/auto-ml-forecasting-github-dau.ipynb

```diff
@@ -0,0 +1,841 @@
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Get the AutoML parent Job"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job_name = returned_job.name\n",
+    "\n",
+    "# Example if providing an specific Job name/ID\n",
+    "# job_name = \"funny_soursop_2zpkp35pdy\"\n",
+    "\n",
+    "# Get the parent run\n",
+    "mlflow_parent_run = mlflow_client.get_run(job_name)\n",
+    "\n",
+    "print(\"Parent Run: \")\n",
+    "print(mlflow_parent_run)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Print parent run tags. 'automl_best_child_run_id' tag should be there.\n",
+    "print(mlflow_parent_run.data.tags)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4.2 Get the AutoML best child run"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get the best model's child run\n",
+    "\n",
+    "best_child_run_id = mlflow_parent_run.data.tags[\"automl_best_child_run_id\"]\n",
+    "print(\"Found best child run id: \", best_child_run_id)\n",
+    "\n",
+    "best_run = mlflow_client.get_run(best_child_run_id)\n",
+    "\n",
+    "print(\"Best child run: \")\n",
+    "print(best_run)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "pd.DataFrame(best_run.data.metrics, index=[0]).T"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4.3 Download the best model locally\n",
+    "\n",
+    "Access the results (such as Models, Artifacts, Metrics) of a previously completed AutoML Run."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create local folder\n",
+    "import os\n",
+    "\n",
+    "local_dir = \"./artifact_downloads\"\n",
+    "if not os.path.exists(local_dir):\n",
+    "    os.mkdir(local_dir)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Download run's artifacts/outputs\n",
+    "local_path = mlflow_client.download_artifacts(\n",
+    "    best_run.info.run_id, \"outputs\", local_dir\n",
+    ")\n",
+    "print(\"Artifacts downloaded in: {}\".format(local_path))\n",
+    "print(\"Artifacts: {}\".format(os.listdir(local_path)))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Show the contents of the MLFlow model folder\n",
+    "os.listdir(\"./artifact_downloads/outputs\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 5 Forecasting using batch endpoint\n",
```

Comment:
Please change the section headers including section numbers to match the energy demand notebook.